src/cdk/rxjs/rxjs.md

```diff
@@ -0,0 +1,36 @@
+### RxJS Usage
+When dealing with RxJS operators, it is best to import the operator functions directly (e.g.
```

Comment:
This should mention that avoiding the patch operators is only recommended when writing a library.